staticcheck.conf

```diff
@@ -0,0 +1,9 @@
+# This is config file for staticcheck.
+# If you need to add ignored checks, pls also add explaination in comments. 
+
+checks = ["all", "-ST1000", "-ST1003", "-ST1016", "-SA1019"]
+
+# ST1000 - Incorrect or missing package comment (non-default)
+# ST1003 – Poorly chosen identifier (non-default)
+# ST1016 – Use consistent method receiver names (non-default)
+# SA1019 - Using a deprecated function, variable, constant or field
```

Comment:
Ignore this check because we are using several deprecated functions currently and I'm not sure how to handle them.

Reply:
let‘s remove the deprecate mark and enbale the check

Reply:
sure.